### DISCUSSION TITLE
GET /sale/offers - filtrowanie po opcjach promowania (badges)

### DISCUSSION THREAD

**Discussion by dawidkozlowski** [allegro/allegro-api#4629] [Suggestion - verification]
Uważam że przydatny byłby filtr po opcjach promowania. Taki jak w managerze allegro
![01](https://user-images.githubusercontent.com/25519114/122564869-227f7a80-d046-11eb-9526-cddc0664c1a6.jpg)
Pomogło by to w przeszukiwaniu większej ilości ofert.

**Comment by nowyyy31** [comment]
I jeszcze przydałoby się informacja taka w API podczas wywoływania polecenia /sale/offers
